parser/src/main/kotlin/com/linkedin/dex/parser/TestMethod.kt

```diff
@@ -9,7 +9,7 @@ import com.linkedin.dex.spec.ClassDefItem
 import com.linkedin.dex.spec.DexFile
 import com.linkedin.dex.spec.MethodIdItem
 
-data class TestMethod(val testName: String, val annotations: List<TestAnnotation>) : Comparable<TestMethod> {
+data class TestMethod(val testName: String, val annotations: MutableList<TestAnnotation>) : Comparable<TestMethod> {
```

Comment:
`MutableList` in a `data class` is a bit sketchy. You shouldn't need this, I believe you can just create a new `TestMethod` when you're adding the inherited annotations